### PR TITLE
902719 - Add the EnableSwiping feature to the UG documentation.

### PR DESCRIPTION
**Pull request by Arunachalam3641**
Add the EnableSwiping feature to the UG documentation.

### CODE REVIEW

File: MAUI/TabView/Swiping.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: About .NET MAUI Tab View (SfTabView) control | Syncfusion
+description: Learn here all about swiping support in Syncfusion .NET MAUI Tab View (SfTabView) control and more.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Swiping.md:4]: @Arunachalam3641, Use the content below,
Learn here all about the swiping support in Syncfusion .NET MAUI Tab View (SfTabView) control and more.

File: MAUI/TabView/Swiping.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+platform: maui
+control: Tab View
+documentation: ug
+---
+
+# Swiping in .NET MAUI Tab View (SfTabView)
+
+The `EnableSwiping` property of `SfTabView` is used to navigate between tab contents using swiping. The default value of `EnableSwiping` is `true.`
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Swiping.md:12]: The `EnableSwiping` property of `SfTabView` allows users to switch between tab contents by swiping. By default, the `EnableSwiping` property is set to `true`.

File: MAUI/TabView/Swiping.md (hunk 1 above)
Comment (Arunachalam3641) [reply MAUI/TabView/Swiping.md]: Yes, I changed the content.

File: MAUI/TabView/Swiping.md (hunk 2 above)
Comment (Arunachalam3641) [reply MAUI/TabView/Swiping.md]: Yes, I changed the content.

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**